sdks/java/io/elasticsearch-tests-5/src/test/java/org/elasticsearch/bootstrap/JarHell.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.elasticsearch.bootstrap;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * We need a real Elasticsearch instance to properly test the IO
+ * (split, slice API, scroll API, ...). Starting at ES 5, to have Elasticsearch embedded,
+ * we are forced to use Elasticsearch test framework. But this framework checks for class duplicates
+ * in classpath and it cannot be deactivated. When the class duplication come from a dependency,
```

Comment:
What are the duplicates?

Reply:
See my comment of the 08/18. Duplicates are 
class: org.apache.maven.surefire.report.SafeThrowable
jar1: /home/echauchot/.m2/repository/org/apache/maven/surefire/surefire-api/2.20/surefire-api-2.20.jar
jar2: /home/echauchot/.m2/repository/org/apache/maven/surefire/surefire-junit47/2.20/surefire-junit47-2.20.jar
Do you want that I put this piece of information in the javadoc?

Reply:
Oh. Is it possible maybe to remove the duplicate classes by shading? If it's not trivial, then yeah, adding something along these lines to javadoc (e.g. say there are conflicts between artifact X and artifact Y) would be sufficient.